smoke_zephyr/utilities.py

```diff
@@ -530,6 +532,52 @@ def is_valid_email_address(email_address):
 	# requirements = re
 	return EMAIL_REGEX.match(email_address) != None
 
+def get_ip_list(ip_network, mask=None):
+	"""
+	Quickly convert an IPv4 or IPv6 network (CIDR or Subnet) to a list
+	of individual IPs in their string representation.
+
+	:param str ip_network:
+	:param str mask:
+	:return: list
+	"""
+	if _its_pyv3:
+		if mask and '/' not in ip_network:
+			net = ipaddress.ip_network("{0}/{1}".format(ip_network, mask))
+			return [host.__str__() for host in net.hosts()]
+		elif '/' not in ip_network:
+			return [str(ipaddress.ip_address(ip_network))]
+		else:
+			net = ipaddress.ip_network(ip_network)
+			return [host.__str__() for host in net.hosts()]
+	elif _its_pyv2:
+		if mask and '/' not in ip_network:
+			net = netaddr.IPNetwork("{0}/{1}".format(ip_network, mask))
+			return [host.__str__() for host in net.iter_hosts()]
+		elif '/' not in ip_network:
+			return [str(netaddr.IPAddress(ip_network))]
+		else:
+			net = netaddr.IPNetwork(ip_network)
+			return [host.__str__() for host in net.iter_hosts()]
+
+def sort_ip_list(ip_list, unique=True):
```

Comment:
I'm going to rename this to `sort_ipv4_list` to be a bit more explicit until we have IPv6 support (if we ever have IPv6 support). You make it really clear in the docs, but I think renaming the function would be helpful as well.